static/app/views/performance/browser/resources/resourceSummaryPage/resourceSummaryTable.tsx

```diff
@@ -0,0 +1,93 @@
+import {Fragment} from 'react';
+
+import GridEditable, {
+  COL_WIDTH_UNDEFINED,
+  GridColumnHeader,
+  GridColumnOrder,
+} from 'sentry/components/gridEditable';
+import {RateUnits} from 'sentry/utils/discover/fields';
+import {useLocation} from 'sentry/utils/useLocation';
+import {useParams} from 'sentry/utils/useParams';
+import {useResourcePagesQuery} from 'sentry/views/performance/browser/resources/utils/useResourcePageQuery';
+import {DurationCell} from 'sentry/views/starfish/components/tableCells/durationCell';
+import {renderHeadCell} from 'sentry/views/starfish/components/tableCells/renderHeadCell';
+import {ThroughputCell} from 'sentry/views/starfish/components/tableCells/throughputCell';
+
+type Row = {
+  'avg(span.self_time)': number;
+  'spm()': number;
+  transaction: string;
+};
+
+type Column = GridColumnHeader<keyof Row>;
+
+function ResourceSummaryTable() {
+  const location = useLocation();
+  const {groupId} = useParams();
+  const {data, isLoading} = useResourcePagesQuery(groupId);
+
+  const columnOrder: GridColumnOrder<keyof Row>[] = [
+    {key: 'transaction', width: COL_WIDTH_UNDEFINED, name: 'Found on page'},
+    {
+      key: 'spm()',
+      width: COL_WIDTH_UNDEFINED,
+      name: 'Throughput',
+    },
+    {
+      key: 'avg(span.self_time)',
+      width: COL_WIDTH_UNDEFINED,
+      name: 'Avg Duration',
+    },
+  ];
+
+  const renderBodyCell = (col: Column, row: Row) => {
+    const {key} = col;
+    if (key === 'spm()') {
+      return <ThroughputCell rate={row[key] * 60} unit={RateUnits.PER_SECOND} />;
+    }
+    if (key === 'avg(span.self_time)') {
+      return <DurationCell milliseconds={row[key]} />;
+    }
+    return <span>{row[key]}</span>;
```

Comment:
m: should we make these links to a transaction like db queries page does it?

Reply:
I'm thinking of doing the same span samples sidebar just like db queries does? Any thoughts on that?

Reply:
yeah works for me!